.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+repos:
+  - repo: https://github.com/psf/black
+    rev: 22.1.0
+    hooks:
+      - id: black-jupyter
+
+  - repo: https://github.com/PyCQA/flake8
+    rev: 4.0.1
+    hooks:
+      - id: flake8
+
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1
+    hooks:
+      - id: isort
+        name: isort (python)
+        args: [--profile black]
```

Comment:
So, `black-jupyter` and `isort` have access to their config files but `isort` doesn't? Or is the `--profile black` option redundant here? Because if `isort` doesn't go looking for what's in the `pyproject.toml` file, then it won't know about the custom line length.

Reply:
`isort` hook actually read the `pyproject.toml` configuration, so I removed the args line, see fe2e49f4a9a4420dd20f518af5df6a18753a4a39.